upstream-sha256: d325ec479a59b78b8bbb4e1a6760df3c03419ea67483581ed21164613a59dca4
actual-sha256: a5873fc843edfc976d4a6dbcb2c602d6074b40441a344d55ff96d8029e1b2214
--- upstream.expected
+++ tolk.actual
@@ -12,16 +12,5 @@
   float val5 = (*(data1_128+(lidx0+64)));
   float val6 = (*(data1_128+(lidx0+80)));
   float val7 = (*(data1_128+(lidx0+96)));
-  *(data0_16+lidx0) = (val0+val2+val3+val4+val5+val6+val7+val1);
+  *(data0_16+lidx0) = (0.0f+val0+val2+val3+val4+val5+val6+val7+val1);
 }
----
-typedef unsigned int uint;
-#define INFINITY (__int_as_float(0x7f800000))
-#define NAN (__int_as_float(0x7fffffff))
-template <class T, class F> __device__ __forceinline__ T tg_bitcast(F v) { union U { F f; T t; }; U u; u.f = v; return u.t; }
-extern "C" __global__ void __launch_bounds__(4) E_4_4(float* data0_16, float* data1_16) {
-  int lidx0 = threadIdx.x; /* 4 */
-  int alu0 = (lidx0<<2);
-  float4 val0 = (*((float4*)((data1_16+alu0))));
-  *((float4*)((data0_16+alu0))) = val0;
-}
